client/src/components/BreadcrumbTrail.tsx

```diff
@@ -31,22 +32,29 @@ export const BreadcrumbTrail = ({
   ascendantObjects,
   parentField,
   isLink,
+  hideParents,
   style
 }: BreadcrumbTrailProps) => {
   const trail = utils.getAscendantObjectsAsList(
     leaf,
     ascendantObjects,
     parentField
   )
+  if (hideParents) {
+    trail.splice(0, trail.length - 1)
+  }
```

Comment:
Only the current displayed task should be removed from the breadcrumb trail. E.g. make task EF 1 selectable, and then look at the matrix. It should show:
```
* EF 1
» 1.1
» 1.1 » 1.1.A
[…etc…]
```